### PR TITLE
Organize Project Assignment

### PR DESCRIPTION
**Pull request by NickJGG**

### CODE REVIEW

**Comment by derickso8** [comment]
Good work Nick.  The readmes, changelog, and license file all look correct. Explanation: Inside the Core folder, which stores most of the project, is the templates folder. This folder stores the html and css of the website for display at certain URLs.

**Comment by nfishel48** [comment]
Looks good ✅

File: README.md [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
+      pip install -r requirements.txt
+      ```
+5. Migrate any changes
+      ```
+      python manage.py migrate
+      ```
+6. Run the server
+      ```
+      python manage.py runserver
+      ```
```
Comment (SuprajaKalva) [inline README.md:39]: Need to add the web URL of the host page that the user uses to access the website

File: README.md [hunk 2]
```diff
@@ -1,21 +1,43 @@
...
 - Pip
 
-# Commands to run the project
-- pip install virtualenv
-- virtualenv env
-- source env/bin/activate
-- pip install -r requirements.txt
-- python manage.py runserver
+### Installation
+
```
Comment (SuprajaKalva) [inline README.md:12]: Might need to give more explicit information about where exactly to run these instructions... Like making sure it is run in the root folder

File: README.md [hunk 3]
```diff
@@ -1,21 +1,43 @@
 # **FILM CLUB**
 
-# Requirements to run this project locally
-- Python 3, the project is being developed with Python 3.8.2
+# Getting Started
+
+### Prerequisites
+
```
Comment (SuprajaKalva) [inline README.md:6]: Where can you get these from? How to install these?

**Comment by ghost** [comment]
I do appreciate the time that was taken to this.  I suggest the merge go ahead.

**Review by SuprajaKalva — COMMENTED** [review]
We still need to add information regarding our standard for everything.

File: README.md (hunk 1 above)
Comment (NickJGG) [reply README.md:39]: (Look below #6)

File: README.md (hunk 2 above)
Comment (NickJGG) [reply README.md:12]: Added

File: README.md (hunk 3 above)
Comment (NickJGG) [reply README.md:6]: Added

**Comment by SuprajaKalva** [comment]
Great job Nick! Thanks for updating all the suggested changes, and the files look great! The core folder like David said stores most of the necessary elements of the project. There is a static subfolder from that root folder that contains all the css / design elements necessary for the project.

**Comment by sjiang1** [comment]
The readme files are very nice!

**Pull request merged**